packages/components/src/components/collapsible/collapsible.jsx

```diff
@@ -1,21 +1,23 @@
 import classNames from 'classnames';
-import React, { useState, Children } from 'react';
+import React, { useState, useEffect, Children } from 'react';
 import { positionPropType } from './utils';
 import ArrowButton from './arrow-button.jsx';
 
 const Collapsible = ({ as, is_collapsed, position = 'top', children }) => {
     const [is_open, expand] = useState(!is_collapsed);
+    const [should_show_collapsible, setShouldShowCollapsible] = useState(false);
     const toggleExpand = () => expand(!is_open);
     const arrow_button = <ArrowButton is_open={is_open} position={position} onClick={toggleExpand} />;
     const CustomTag = as || 'div';
+    useEffect(() => setShouldShowCollapsible(Children.toArray(children).some(({ props }) => 'collapsible' in props)));
     return (
         <CustomTag
             className={classNames('dc-collapsible', {
                 'dc-collapsible--is-expanded': is_open,
                 'dc-collapsible--is-collapsed': !is_open,
             })}
         >
-            {position === 'top' && arrow_button}
+            {should_show_collapsible && position === 'top' && arrow_button}
```

Comment:
will this always be `true`? because we're always passing `collapsible` children as defined in `screen-small.jsx`

Reply:
unless we render those `collapsible` children conditionally.. something like 
```
<Collapsible>
      {isVisible && <div collapsible .. />}
</Collapsible>
```